alena/hs_err_pid16108.log

```diff
@@ -0,0 +1,197 @@
+#
+# There is insufficient memory for the Java Runtime Environment to continue.
```

Comment:
this file should not be here

Reply:
agree with you , it was added by mistake. cancel not helped: everything was pushed. Sorry :)